addon/helpers/d3-tick-format.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import { timeFormat } from 'd3-time-format';
+
+export function d3TickFormat([tickFormat]) {
```

Comment:
@ivanvanderbyl thanks - this is very nice, I decided to go a different direction for now, and use the `tickValue` method on the axis - you can see what I did here: https://github.com/LocusEnergy/ember-sparkles/blob/9e483374b45444ce1bc1b7ee284a540fbb2f9156/addon/helpers/ember-sparkles/axis.js#L13-L15

I found this idea here - http://bl.ocks.org/mbostock/3212294

Taras and I discussed this - we plan to handle this a bit more programmatically when we implement axes as contextual components (similarly to what you are doing in Maximum Plaid!)